app/replica.go

```diff
@@ -84,29 +84,46 @@ func CheckReplicaState(frontendIP string, replicaIP string) (string, error) {
 }
 
 func AutoConfigureReplica(s *replica.Server, frontendIP string, address string, replicaType string) {
-checkagain:
-	state, err := CheckReplicaState(frontendIP, address)
-	logrus.Infof("Replicastate: %v err:%v", state, err)
-	if err == nil && (state == "" || state == "ERR") {
+	var (
+		err   error
+		state string
+	)
+	for {
+		state, err = CheckReplicaState(frontendIP, address)
+		logrus.Infof("Replicastate: %v err:%v", state, err)
+
+		if err != nil {
+			logrus.Infof("checkReplicaState failed, err:%v retry after 5sec..", err)
+			time.Sleep(5 * time.Second)
+			continue
+		}
+		if state == "WO" || state == "ERR" {
```

Comment:
we want to autormreplica for WO also?
Please add comments if we need to this, and also, add a log that autormreplica is being attempted